llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -995,34 +995,36 @@ static bool isGlobalVarSummary(const ModuleSummaryIndex &Index,
   return false;
 }
 
+// Return the number of global summaries and record the number of function
+// summaries as output parameter.
 static unsigned numGlobalVarSummaries(const ModuleSummaryIndex &Index,
                                       FunctionImporter::ExportSetTy &ExportSet,
-                                      unsigned &DefinedGVS,
                                       unsigned &DefinedFS) {
-  DefinedGVS = 0;
+  unsigned NumGVS = 0;
   DefinedFS = 0;
   for (auto &VI : ExportSet) {
-    if (isGlobalVarSummary(Index, VI.getGUID())) {
-      ++DefinedGVS;
-    } else
+    if (isGlobalVarSummary(Index, VI.getGUID()))
+      ++NumGVS;
+    else
       ++DefinedFS;
```

Comment:
Can't DefinedFS be deduced from ExportSet.size() - NumGVS?

Reply:
Yes the number of function summaries could be deduced. I simplified the function.